index.html

```diff
@@ -3,43 +3,95 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>State reserves</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
       text-align: center;
     }
   </style>
 </head>
-<body>
+        <style>
+table {
+    border-collapse: collapse;
+    width: 100%;
+}
+
+th, td {
+    text-align: left;
+    padding: 8px;
+}
+
+tr:nth-child(even){background-color: #f2f2f2}
+
+th {
+    background-color: #4CAF50;
+    color: white;
+}
+</style>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <h1>How long could your state survive on current reserves?</h1>
+      <p>Total reserves in millions and days' supply of general fund operations.</p>
     </div>
     <table class="table">
       <thead>
         <tr>
-          <th>Fruit</th>
-          <th>Price</th>
-          <th>Quantity</th>
+          <th>State</th>
+          <th>Days' worth GF reserves</th>
+          <th>Total reserves in millions</th>
         </tr>
       </thead>
       <tbody>
         <tr>
-          <td>Apples</td>
-          <td>$0.55</td>
-          <td>5</td>
+          <td>Alabama</td>
+          <td>29.7</td>
+          <td>627</td>
         </tr>
         <tr>
-          <td>Oranges</td>
-          <td>$0.80</td>
-          <td>10</td>
+          <td>Alaska</td>
+          <td>447.0</td>
+          <td>7365</td>
         </tr>
         <tr>
-          <td>Pears</td>
-          <td>$0.95</td>
-          <td>3</td>
+          <td>Arizona</td>
+          <td>29.4</td>
+          <td>748</td>
```

Comment:
Would be useful to have a $ sign here so that it's more obvious what the figures are.